apps/ui/src/networks/types.ts

```diff
@@ -238,7 +239,8 @@ export type NetworkActions = ReadOnlyNetworkActions & {
     space: Space,
     networkId: NetworkID,
     delegatee: string,
-    delegationContract: string
+    delegationContract: string,
+    delegationType: DelegationType
```

Comment:
nit. I feel like it would look better if delegationType was earlier as it's more general (before delegatee).